<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>20</number>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>Количество мест для абитуриентов, сдавших экзамены в другом вузе:</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="labelWarning">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>320</width>
              <height>70</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <underline>true</underline>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Мест нет.
Вы можете добавить только абитуриентов, сдавших экзамены в этом вузе.</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>250</height>
        </size>
       </property>
       <property name="title">
        <string>Добавление абитуриента в список</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>35</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>35</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBoxThisUniversity">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Сдал экзамены в этом вузе</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Фамилия:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSurname">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Имя:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditName">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Отчество:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditPatronim">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="checkBoxAttendedExam">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Присутствовал на экзамене</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Оценки:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLineEdit" name="lineEditMark1">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>29</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>10</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditMark2">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>29</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>10</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditMark3">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>29</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAdd">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Добавить абитуриента в список</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Поиск по фамилии </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Фамилия:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditFindBySurname">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonInfo">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>150</height>
        </size>
       </property>
       <property name="title">
        <string>Функции:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButtonDelete">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Удалить абитуриентов, отсутствующих на экзамене </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonHighAveragePoint">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Показать абитуриентов, средний балл которых
выше среднего балла по универститету</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonShowAll">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Показать весь список</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
